book/views.py

```diff
@@ -1,3 +1,23 @@
-from django.shortcuts import render
+from rest_framework import viewsets
+from rest_framework.response import Response
 
-# Create your views here.
+from book.models import Book
+from rest_framework.decorators import action
+from book.serializers import BookSerializer
+
+
+class BookViewSet(viewsets.ModelViewSet):
+    queryset = Book.objects.all()
+    serializer_class = BookSerializer
+
+    @action(detail=True, methods=['get'], url_path='detail')
```

Comment:
Why do use @action here? Do we need additional endpoint? Also fix flake8 and quotes